charts/tidb-cluster/templates/tikv-importer-service.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.importer.create }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "cluster.name" . }}-importer
+  labels:
+    app.kubernetes.io/name: {{ template "chart.name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: importer
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
+spec:
+  clusterIP: None
+  ports:
+  - name: importer
+    port: 20170
```

Comment:
We use port 8287 for importer.

Reply:
Then what is this port for? https://github.com/tikv/importer/blob/master/etc/tikv-importer.toml#L10

Reply:
The sample config file is not yet updated 🤣